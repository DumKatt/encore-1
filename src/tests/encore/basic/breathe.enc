class Foo
  def foo() : void {
    print "out of air!";
    breathe;
    print "that's better"
  }

class Main
  def main() : void
    (new Foo).foo()
