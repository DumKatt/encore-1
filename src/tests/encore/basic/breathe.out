out of air!
that's better
